scripts/presubmit-tests.sh

```diff
@@ -304,6 +304,15 @@ function main() {
       echo ">> docker version"
       docker version
     fi
+    # TODO(chaodaiG): node/pod names are missing after migrating from bootstrap to podutil,
+    # report it here with the same logic as in bootstrap,
+    # until we figure out a way of reporting it in podutil
+    # see https://github.com/kubernetes/test-infra/blob/09bd4c6709dc64308406443f8996f90cf3b40ed1/jenkins/bootstrap.py#L588
+    echo ">> node name"
+    echo "$(curl -H "Metadata-Flavor: Google" 'http://169.254.169.254/computeMetadata/v1/instance/name' 2> /dev/null)"
```

Comment:
Where does the IP come from? Will it change without notice and break the log?

Reply:
It comes from bootstrap.py, which was how node name got injected before. The link above the line points directly to that line